src/containers/Toolbar/index.tsx

```diff
@@ -42,11 +41,13 @@ export const Toolbar = (props: IToolbarProps) => {
         testId="clear-network-table"
         className="-mr-3"
       />
-      <Textfield
+      <input
```

Comment:
What was the reason for moving from Textfield > input? Could we achieve the same result with a component to avoid more global styles?

Using one component means functional changes can be reflected across the app. Using an input means future changes would need to be repeated in many places.

Reply:
I think it's probably just an opinion, after my changes TextField just felt like it had no actual _code-behaviour_ (just _style-behaviour_) and so it made sense to remove for now (and the tailwind code re-use mechanism seemed like the right way to share style-behaviours).

Reply:
Switched back to a component, and removed the `@apply`